distribution/docs/src/main/resources/_contents/_sources/solr-catalog-provider-contents.adoc

```diff
@@ -1,43 +1,25 @@
+=== Solr Catalog Types
+The ${branding} supports 3 Solr Catalog Types:
 
-==== Catalog Solr Embedded Provider
+. Solr Catalog Embedded Provider - included with the ${branding}, useful for testing and demonstration pruposes
+. Solr Catalog External Provider - single Solr Server instance
+. SolrCloud - cluster of standalone Solr Servers
+
+==== Solr Catalog Embedded Provider
 
 The Solr Catalog Embedded Provider is an embedded, local Solr Server instance used in conjunction with an Apache Solr Server data store.
-It is a local instance that is a lightweight Catalog provider solution. However, it does not provide a Solr Admin GUI or a "REST-like HTTP/XML and JSON API." If that is necessary, see Standalone Solr Server.
+It is a local instance that is a lightweight Catalog provider solution.
 
 Use the local, embedded Solr Catalog Provider when only one ${branding} instance is necessary and scalability is not an issue.
 The local, embedded Solr Catalog Provider requires no installation and little to no configuration.
 It is great for demonstrations, training exercises, or for sparse querying and ingesting.
 
-===== Installing Solr Catalog Embedded Provider
-
-The Solr Catalog Embedded Provider is not installed by default with a standard installation.
-
-To install:
-
-. Navigate to the ${ddf-solr} application in the ${admin-console}.
-. Select *Features*.
-. Uninstall the `catalog-solr-external-provider` feature.
-. Install the `catalog-solr-embedded-provider`.
-
-==== Configuring Embedded Solr Server and Solr Catalog Provider
-
-No configuration is necessary for the embedded Solr Server and the Solr Catalog Provider.
-The standard installation described above is sufficient.
-When the `catalog-solr-embedded-provider` feature is installed, it stores the Solr index files to `<DISTRIBUTION_INSTALLATION_DIRECTORY>/data/solr` by default.
-A user does not have to specify any parameters and the `catalog-solr-embedded-provider` feature contains all files necessary for Solr to start the server. 
-
-However, this component _can_ be configured to specify the directory to use for data storage. 
-
-The configurable properties for the SCP are accessed from the *Catalog Embedded Solr Catalog Provider* configurations in the ${admin-console}.
-
 [TIP]
```

Comment:
The new Catalog Solr Provider is the default.

https://github.com/codice/ddf/blob/master/catalog/solr/solr-app/src/main/resources/features.xml#L49

But it looks like I forgot to update the metatype to remove the `Catalog External` part before `Solr Catalog Provider`.  That would make it more consistent with the feature and bundle name.

https://github.com/codice/ddf/blob/master/catalog/solr/catalog-solr-provider/src/main/resources/OSGI-INF/metatype/metatype.xml#L18

https://github.com/codice/ddf/blob/master/catalog/solr/catalog-solr-provider/pom.xml#L25